contracts/solve/test/Inbox_cancel.t.sol

```diff
@@ -55,7 +54,7 @@ contract SolveInbox_cancel_Test is InboxBase {
         // cannot cancel accepted request
         vm.prank(solver);
         inbox.accept(id);
-        vm.expectRevert(SolveInbox.RequestStateInvalid.selector);
+        vm.expectRevert(SolveInbox.NotPendingOrRejected.selector);
```

Comment:
@Zodomo went with more explicit error names. makes  reversions and tests clearer IMO